src/blueprints/tv2/tv2-action-service.ts

```diff
@@ -93,14 +123,21 @@ export class Tv2ActionService implements BlueprintGenerateActions {
   private getPieceTypeFromActionManifest(actionManifest: ActionManifest): Tv2PieceType {
     switch (actionManifest.actionId) {
       case 'select_full_grafik': {
-        return Tv2PieceType.CAMERA
+        return Tv2PieceType.GRAPHICS
```

Comment:
We have a PieceType called `GRAPHICS` and another PieceType called `OVERLAY_GRAPHICS`.
If `GRAPHICS` is only used for Fullscreen, then it would be nice if the name reflected that.

Reply:
I see what you mean. At the moment it appears to only be used for Full Graphics.

I am open for chainging it, however before i do, i would like to know if such a change should also be done in the frontend code, as the Tv2PieceType is used for type on metadata.

Reply:
With our current setup, then yes, that changed should also be made in the frontend.